src/commitizen/configLoader.js

```diff
@@ -1,10 +1,13 @@
-import { loader } from '../configLoader';
+import { cosmiconfig } from "cosmiconfig";
 
 export { load };
 
-// Configuration sources in priority order.
-var configs = ['.czrc', '.cz.json', 'package.json'];
+async function load(config, cwd) {
+  const explorer = cosmiconfig("commitizen", {
+    packageProp: "config.commitizen",
+    searchPlaces: [".czrc", ".cz.json", "package.json"],
+    searchFrom: cwd,
```

Comment:
By default, this is the current working directory. Not sure what the `cwd` argument represents here (maybe it's just for tests?) so I made this option explicit just in case.